content/lua/concepts/coroutines/coroutines.md

```diff
@@ -0,0 +1,34 @@
+---
+Title: 'Coroutines' # Required; the file name should be the same as the title, but lowercase, with dashes instead of spaces, and all punctuation removed
+Description: 'Coroutines allow the execution of multiple tasks at the same time without yielding by using separate threads.' # Required; ideally under 150 characters and starts with a noun (used in search engine results and content previews)
+Subjects: # Please only use Subjects in the subjects.md file (https://github.com/Codecademy/docs/blob/main/documentation/subjects.md). If that list feels insufficient, feel free to create a new Subject and add it to subjects.md in your PR!
+  - 'Computer Science'
+Tags: # Please only use Tags in the tags.md file (https://github.com/Codecademy/docs/blob/main/documentation/tags.md). If that list feels insufficient, feel free to create a new Tag and add it to tags.md in your PR!
+CatalogContent: # Please use course/path landing page slugs, rather than linking to individual content items. If listing multiple items, please put the most relevant one first
+  - 'learn-lua'
+---
+
+**Coroutines** allow the execution of multiple tasks at the same time without yielding by using separate threads.
```

Comment:
I think this sounds like multi-threading, which isn't really what coroutines are. From what I've read, they allow for concurrency not parallelism. I think this definition should be rephrased a bit to be closer to the description suggested above.